core/src/main/scala/org/apache/spark/sql/delta/Checkpoints.scala

```diff
@@ -255,7 +258,7 @@ object Checkpoints extends DeltaLogging {
 
     val (factory, serConf) = {
       val format = new ParquetFileFormat()
-      val job = Job.getInstance()
+      val job = Job.getInstance(hadoopConf)
```

Comment:
Fixing a bug that's not using the correct Hadoop configuration. The new unit test requires this fix.